webapp/src/app/shared/phaser/entities/cc-entity.ts

```diff
@@ -31,6 +31,12 @@ export interface AttributeValue {
 	[key: string]: any;
 }
 
+export interface DetailSettings {
+	mapId: number;
```

Comment:
All uses of `mapId` act as if it were optional. One even does `Partial<DetailSettings>` just for that.

Reply:
I moved the call to `map.getUniqueMapid()` to better reflect what i meant with that. It basically says imported map can contain entities without a mapId, but the internal representation of entities always has a mapId.

It is currently unused but having the type is still better than using `any`